plugins/newrelic-dashboard/src/api/NewRelicDashboardClient.ts

```diff
@@ -27,21 +27,27 @@ import { ResponseError } from '@backstage/errors';
 
 export class NewRelicDashboardClient implements NewRelicDashboardApi {
   private readonly discoveryApi: DiscoveryApi;
+  private readonly identityApi: IdentityApi;
```

Comment:
Think that it actually makes sense to use the `fetchApi` here, which has the ability to automatically inject the headers necessary from the `identityApi` for you, so you won't need to do anything else.

It should just replace the `fetch` call, and I think looking at the code it should be a drop in replacement.

Reply:
Yep ! That's a fantastic suggestion  Thanks  a lot @benjdlambert   I have just added `fetchApi` and committed code https://github.com/backstage/backstage/pull/10754/commits/7eee3434aacd2a72d1ac5823b3f23a44db92f76a  , please have a look and let me know for any changes